app/src/main/java/com/kickstarter/ui/views/compose/checkout/CollectionPlan.kt

```diff
@@ -0,0 +1,243 @@
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.RadioButton
+import androidx.compose.material.RadioButtonDefaults
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.selected
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.kickstarter.R
+import com.kickstarter.ui.compose.designsystem.KSTheme
+import com.kickstarter.ui.compose.designsystem.KSTheme.colors
+import com.kickstarter.ui.compose.designsystem.KSTheme.dimensions
+import com.kickstarter.ui.compose.designsystem.KSTheme.typography
+
+@Preview(showBackground = false, name = "Eligible - Pledge in Full Selected")
+@Composable
+fun PreviewPledgeInFullSelected() {
+    KSTheme {
+        CollectionPlan(isEligible = true, initialSelectedOption = "Pledge in full")
+    }
+}
+
+@Preview(showBackground = false, name = "Eligible - Pledge Over Time Selected")
+@Composable
+fun PreviewPledgeOverTimeSelected() {
+    KSTheme {
+        CollectionPlan(isEligible = true, initialSelectedOption = "Pledge Over Time")
+    }
+}
+
+@Preview(showBackground = false, name = "Not Eligible")
```

Comment:
@jlplks nitpick here can we add the same previews as well for the darktheme?

Reply:
![Screenshot 2024-11-12 at 3 39 48 p m](https://github.com/user-attachments/assets/83128daa-3d05-447c-81ed-d90f2290d442)